src/main/java/se/iths/librarysystem/dto/Book.java

```diff
@@ -8,13 +8,13 @@
     private String title;
     private String subtitle;
     private String edition;
-    private Date printed;
+    private String printed;
```

Comment:
```suggestion
   @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yy hh:mm:ss")
    private Date printed;
```

If we decide to use Date as a variable type, we can annotate the field using `@JsonFormat` to choose the date format in the JSON response body.